src/lib/hydrate.js

```diff
@@ -1,16 +1,17 @@
-'use strict';
-
-const Items = require('warframe-items');
-const data = require('warframe-worldstate-data');
-const flatCache = require('flat-cache');
-const path = require('path');
-const fetch = require('node-fetch');
-const Logger = require('./logger');
+import Items from 'warframe-items';
+import data from 'warframe-worldstate-data';
+import flatCache from 'flat-cache';
+import path, { dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import Logger from './logger.js';
+import Settings from './settings.js';
 
 const {
   wfInfo: { filteredItems: filteredItemsSrc, prices: pricesSrc },
```

Comment:
there's no naming collision, is this renaming so we get an association with how raw and src are related data?

Reply:
yeah, just to point out that it's not data, given this is a setting, but rather that it's a url